src/tests/DataLayer.test.js

```diff
@@ -116,6 +116,26 @@ test('add function: simple', () => {
   expect(mockCallback.mock.calls.length).toBe(1);
 });
 
+test('add function: add listener with adobeDataLayer:change event', () => {
```

Comment:
Isn't this test just a duplicate of another test called `check dataLayer change event was executed`?

Reply:
sorry, didn't see your question before merging. This test is a little bit different as the event listener registration happens within a function push. So it combines 2 features.

Reply:
It would be good to rename it then to indicate the difference. I will do it with additional tests I will provide